TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/ChangeRequests/EditChangeRequest/Index.cshtml.cs

```diff
@@ -31,6 +32,16 @@ public void OnGet()
     {
     }
 
+    public async Task<IActionResult> OnGetEvidenceAsync()
+    {
+        using var stream = await fileService.OpenReadStreamAsync(Evidence!.FileId);
+        using var ms = new MemoryStream();
+        await stream.CopyToAsync(ms);
+        var bytes = ms.ToArray();
+
+        return File(bytes, Evidence.MimeType);
```

Comment:
Isn’t there an overload of this that takes a Stream (to avoid pulling the whole thing into memory at once)?

Reply:
ah so there is!